docs/cron-workflows.md

```diff
@@ -39,7 +39,7 @@ In the above example it would be similar to `test-cron-wf-tj6fe`.
 
 You can use `CronWorkflow.spec.workflowMetadata` to add `labels` and `annotations`.
 
-### `CronWorkflow` Options
+### `CronWorkflow` Spec
```

Comment:
I'm surprised this doesn't fail `markdownlint`, but this is the existing `h2` already. With this change, this section is now "## `CronWorkflow` Spec" -> "### `CronWorkflow` Spec"

Reply:
Yes. I've reverted this but it's still a bad title.

Reply:
Calling it "Spec" wouldn't quite be correct either bc it doesn't include all the fields, such as the ones above and others. The closest thing might be to call it "Fields", as in to match the "Field Reference". Though I honestly feel this duplicates the existing section there that we could just link to instead: https://argo-workflows.readthedocs.io/en/latest/fields/#cronworkflowspec